### PR TITLE
Added test case #19

### PR DESCRIPTION
**Pull request by bridanrod**

### CODE REVIEW

File: cypress/e2e/Tests/ViewAndCartBrandProducts.cy.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import homePage from "../PageObjects/HomePage";
+import ProductDetailsPage from "../PageObjects/ProductDetailsPage";
```
Comment (andresclavijov) [inline cypress/e2e/Tests/ViewAndCartBrandProducts.cy.js]: I would change _ProductDetailsPage_ to _productDetailsPage_

File: cypress/e2e/Tests/ViewAndCartBrandProducts.cy.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        let selectedBrand;
+        cy.url().then(url => {
+            selectedBrand = url.split('/').pop();
+            productsPage.clickViewProductOfFirstItem();
+            ProductDetailsPage.verifyProductBrand(selectedBrand);
+        });
+        
+    });
+
+    it('When I click on another brand', () => {
```
Comment (andresclavijov) [inline cypress/e2e/Tests/ViewAndCartBrandProducts.cy.js]: When writing a test case on Gherkin, try to maintain the order of the keywords (Given > When > Then) and use (if possible!) only one 'When' sentence with no 'and' following it

**Review by andresclavijov — APPROVED** [review]
Great job! 🥳

File: cypress/e2e/Tests/ViewAndCartBrandProducts.cy.js (hunk 1 above)
Comment (bridanrod) [reply cypress/e2e/Tests/ViewAndCartBrandProducts.cy.js]: Fixed! 🙌🏼

File: cypress/e2e/Tests/ViewAndCartBrandProducts.cy.js (hunk 2 above)
Comment (bridanrod) [reply cypress/e2e/Tests/ViewAndCartBrandProducts.cy.js]: Fixed! 🙌🏼

**Pull request merged**